src/fsipy/automatedPreprocessing/automated_preprocessing.py

```diff
@@ -439,6 +444,14 @@ def run_pre_processing(input_model, verbose_print, smoothing_method, smoothing_f
             # Write mesh in VTU format
             write_polydata(remeshed_surface, file_name_surface_name)
             write_polydata(mesh, file_name_vtu_mesh)
+
+        # Write the mesh ID's to parameter file
+        parameters["solid_side_wall_id"] = solid_side_wall_id
+        parameters["interface_fsi_id"] = interface_fsi_id
+        parameters["interface_outer_id"] = interface_outer_id
+        parameters["volume_id_fluid"] = volume_id_fluid
+        parameters["volume_id_solid"] = volume_id_solid
```

Comment:
If we follow the naming above, it might be better to call them `volume_fluid/solid_id`

Reply:
Perhaps even `fluid_volume_id` and `solid_volume_id`? What do you like better?

Reply:
`fluid_volume_id` and `solid_volume_id` sound great!